V 1.0 (#8)
Zubax/Komar: manufacturing/readme.md

> ## Zubax Komar power stage manufacturing instructions

Reviewer: ```suggestion
# Zubax Komar power stage manufacturing instructions
```

The manufacturing documents package can be found attached to the releases in this repository. It contains following entities:

* Gerber - all the necessary gerber files are contained here.
* Assembly - assembly drawings (in PDF format) and centroid files are contained here.
* BOM - a bill of materials in a form of spreadsheet.
* PCB stack - a file containing description of the PCB stack and layers.
* Wires drawing.

It should be noted that Komar is a highly reliable device and consists of AEC-Q qualified components, thus any changes to BOM are **highly undesirable**.

The device assembly involves soldering wires. Considering that Digikey has a very limited choice of wires and the overall impracticality of the idea of buying wires on Digikey it is recommended to source the wires from local suppliers. The criteria are:

* Wire material - copper
* Wire type - multi stranded

> * AWG (12 and 14 AWG are used in the design)

Reviewer: Давайте-ка использовать СИ https://www.rapidtables.com/calc/wire/awg-to-mm.html

Author: +

* Color (red and black are used in the design)
* Insulation material (silicone is highly recommended for its outstanding thermal resistance, although PVC may be used as well)

> PCB properties:

Reviewer: Вы тут заголовок хотели сделать?

Author: Точно

> The PCB is 4 layer PCB with 1 oz copper on all layers. Its dimensions is a circle with a diameter of 58 mm. PCB Total thickness is 1.58 - 1.6 mm. Solder Mask color should be red, silkscreen color - white. 

Reviewer 2: ```suggestion
The PCB is 4 layer PCB with 35 um copper on all layers. Its dimensions is a circle with a diameter of 58 mm. PCB Total
 thickness is 1.58 - 1.6 mm. Solder Mask color should be red, silkscreen color - white. 
```

> Copper layer order is:
> 
> 1. copper_top_l1
> 
> 2. copper_inner_l2
> 
> 3. copper_inner_l3
> 
> 4. copper_bottom_l4

Reviewer: ```suggestion
The copper layers are ordered as follows:

1. copper_top_l1
2. copper_inner_l2
3. copper_inner_l3
4. copper_bottom_l4
```

>    ### Assembly

Reviewer: ```suggestion
## Assembly
```

There are several steps to assemble the PCB:

1. **SMD assembly**. After SMD assembly the board should look like this:

   

<img src="pics/1.jpg" width="500"> 

<img src="pics/2.jpg" width="500">

2. **Capacitors and pin headers soldering**.

   C1 and C2 radial electrolytic should be soldered sideways as in picture below. **Observe the polarity!**

   <img src="pics/3.jpg" width="500">

   3. **Wires soldering.** There are total 5 wires to be soldered to the PCB: 2 power supply wires (20 cm long each, one red and one black, 12 AWG cable) and 3 phase wires (20 cm long each, one red and one black, 14 AWG cable). Power supply wires’ ends that are not soldered to the PCB should be stripped and tinned (approx. 5 mm). Phase wires’ ends that are not soldered to the PCB should not be stripped. For better wires representation please refer to wires drawing in the manufacturing doc pack. The wires should be soldered as shown in the pictures below.

      **Phase wires:**

      <img src="pics/4.jpg" width="500">

      **Power wires:**

      <img src="pics/5.jpg" width="500">

      <img src="pics/6.jpg" width="500">

> 4. **Conformal coating.** The PCBs should be covered with conformal coating entirely excluding pin headers on the top (CON100, CON101, CON102, CON103, CON104, CON105) and connectors on the bottom side of the PCB (CON1, CON3, CON4, CON5, CON6, CON7). 

Reviewer: - [x] О приклейке конденсаторов ничего не сказано.
- [x] Я не нашёл в BOM проводов, клея для конденсаторов, конформного покрытия, винтов и проч.
- [ ] Чертёж из Construction and usage в главном README не помешает привести и в сборочной документации, мне кажется.

Author: Чертеж мы приводить не станем. Вместо этого я запилю видео по сборке, вставим его.

Author: Я добавил эту короткую видеоинструкцию. Я знаю, что в ней не хватает пояснительного текста и, возможно, пары компонентов: световода и защитной крышки для разъемов. 
Я не уверен насчет того, что эти компоненты могут и должны использоваться. Понять это мы сможем после того, как соберем хотя бы одно устройство живьем. Тогда же и будут добавлены пояснения в видео, т.к. это довольно муторный процесс, который не хочется переделывать.

   <img src="pics/4.png" width="1000">

5. **Phase connector mounting**. Phase connector consists of a plastic part with epoxy-glued 4.5 mm female bullet connectors with pre-formed and pre-soldered solid wires (12 AWG solid wire).
   
   <img src="pics/5.png" width="1000">

   It should be soldered to the PCB so that bottom surface of the plastic part rests on the PCB (see the pic below).

   <p align="center">
   <img src="pics/6.png" alt="" width="500"/>
   </p>
   
   **Conformal coating.** The PCBs should be covered with conformal coating (see BOM table below for the recommended partnumber) entirely excluding and connectors on the bottom side of the PCB (CON1, CON3, CON4, CON5, CON6, CON7) and phase connector's bullet connectors inner surfaces.

6. **Housing assembly**.  At this step the PCB is placed into the housing. For detailed instructions please watch the video below. 

>    [![Komar housing assembly instruction](http://img.youtube.com/vi/qDjCuD04zwk/0.jpg)](http://www.youtube.com/watch?v=qDjCuD04zwk "Komar housing assembly instruction")

Reviewer: Видео по ссылке, если вы считаете его полезным, нужно переименовать, чтобы было понятно, что речь идёт о версии 1.0.

Author: готово

| PCB spacers                                                  | Spacers that are placed between the PCB and the housing.     | 3d printed parts. ABS plastic.                               |
| Thermal pad                                                  | Provides good thermal contact between the PCB and the bottom part of the housing. | 1 mm thick thermal pad from arctic cooling.                  |
| Motor screws                                                 | Are used for attaching the motor to the top part of the housing. | 4 pcs 92095A188 from mcmaster.com (M4 ISO 7380, 6 mm length screws). |
| Connectors protection cap                                    | Provides some level of protection for interface connectors and may act as a guide for power wires. | 3d printed plastic part. ABS plastic.                        |
| Cap screws                                                   | Are used to hold connectors protection cap in place          | 3 pcs screws 91290A099 from mcmaster.com (M2.5 DIN912 4 mm length screws). |

> | Mounting screws                                              | Are used to mount the assembled Komar in place               | 4 pcs screws 91292A107 from mcmaster.com (M4 DIN 912 6 mm length screws) |

Reviewer: - BOM не согласуется со сборочной схемой.
- Средняя колонка не нужна, объедините её с Description.
- Если деталь стандартная, не вижу смысла указывать поставщика, это добавляет работы по поддержке и актуализации документации.

Author: Относитесь к упоминанию mcmaster не как к указанию поставщика, а как к странице продукта на digikey, у них очень хороший каталог всех винтиков и гаек на свете. Такой артикул полезен, и удалять его не следует. 
С какой сборочной схемой не согласуется BOM? Что это?

Reviewer 2: Я лишь замечу, что в нашем случае "сборочный цех" находится в Эстонии. Потому логичным будет в наших механических изделиях использовать крепеж, который легко достать в Эстонии. Я предлагаю использовать [Baltic Bolt](https://www.balticbolt.ee/ProductGroups/Details/6). Там представлена вся нужная нам номенклатура. 
Что касается сборочных чертежей, то там должен быть просто указан тип крепежа по соответствующему стандарту. К примеру, M3 x 16 (ISO4762). А в BOM указан конкретный PN поставщика, например, 047628100030016 (Baltic Bolt).

Author: Никаких артикулов частных поставщиков крепежа в BOM решено не иметь.
